kusto/ingest/internal/resources/resources.go

```diff
@@ -46,7 +46,7 @@ type URI struct {
 // parse parses a string representing a Kutso resource URI.
 func parse(uri string) (*URI, error) {
 	// Regex representing URI that is expected:
-	// https://(\w+).(queue|blob|table).core.windows.net/([\w,-]+)\?(.*)
+	// https://(\w+).(queue|blob|table).[\w.]+/([\w,-]+)\?(.*)
```

Comment:
This comment is no longer relevant

Reply:
Why? we still check the rest of the uri, I changed the regex to fit any domain.